cli_helpers/tabular_output/output_formatter.py

```diff
@@ -209,11 +209,13 @@ def format_output(data, headers, format_name, **kwargs):
 for tabulate_format in tabulate_adapter.supported_formats:
     TabularOutputFormatter.register_new_formatter(
         tabulate_format, tabulate_adapter.adapter,
-        tabulate_adapter.preprocessors,
+        tabulate_adapter.preprocessors +
+        (tabulate_adapter.style_output_table(tabulate_format),),
```

Comment:
If you wanted to, you could pass `format_name` in via [`format_output`](https://github.com/dbcli/cli_helpers/pull/38/files#diff-54f002145bfa7a641b8eccca6ce5032aR146) so you wouldn't have to handle the different formats here. It's fine as is, too, though.